desktop/app/store-helper.js

```diff
@@ -20,7 +20,10 @@ export default function (mainWindow) {
     subscribeStoreSubscribers.push({sender, selector})
 
     try {
-      sender.send('stateChange', selector ? selector(store) : store)
+      const newStore = selector ? selector(store) : store
```

Comment:
Just bailing if the selector gives us a null. Really we should fix this upstream but this seems safe for now